config/locales/en/course/assessment/answer/text_response_comprehension_auto_grading.yml

```diff
@@ -4,8 +4,16 @@ en:
       answer:
         text_response_comprehension_auto_grading:
           explanations:
+            point: '<u>Part (%{index})</u>'
```

Comment:
Does this work without adding `html` to the key?

Reply:
Nope, because the answer explanation is an array of strings

Reply:
I have added `_html` to the end of the key since it was explicitly mentioned in the Rails I18n API.